core/controllers/ingest.py

```diff
@@ -43,6 +43,10 @@ def ingest(body):
     if validation_failures:
         raise ValidationError(validation_failures=validation_failures)
 
+    # wipe db
+    db.drop_all()
+    db.create_all()
```

Comment:
This seems quite dangerous, can we add a TODO to remove this before we go live?

Reply:
added this